xml/depl_nodes.xml

```diff
@@ -1773,8 +1764,8 @@ getent group postgres | cut -d: -f3</screen>
     </term>
     <listitem>
      <para>
-      Enables the possibility to extract tar files into a swift account and
-      to delete multiple objects or containers with a single request. See
+      Extract tar files into a Swift account, and
```

Comment:
Stefan just suggested to replace 'tar file' with 'TAR archive' because they are synonyms anyway...